config/seckit.settings.yml

```
@@ -0,0 +1,56 @@
+_core:
+  default_config_hash: x6bhN6WZwfVUI_LLMvRJIUW_2c26VTaBozbfXmJWmro
+seckit_xss:
+  csp:
+    checkbox: true
+    vendor-prefix:
+      x: true
+      webkit: false
+    report-only: false
+    default-src: "'self'"
+    script-src: "'self' 'unsafe-inline' 'unsafe-eval' fonts.googleapis.com www.gstatic.com https://*.google.com https://*.googletagmanager.com cdn.jsdelivr.net *.google-analytics.com https://cdnjs.cloudflare.com"
+    object-src: "'none'"
+    style-src: "'self' 'unsafe-inline' fonts.googleapis.com https://cdnjs.cloudflare.com"
+    img-src: "'self' data: https://*"
+    media-src: "'none'"
+    frame-src: "'self' https://*.mapbox.com"
```

```suggestion
    frame-src: "'self' https://*.mapbox.com https://*.youtube.com https://*.youtu.be"
```
We need to add `https://*.youtube.com` and possibly `https://*.youtu.be`. For example for the video at the bottom of https://reliefweb.int/topic/iasc-task-force-5-localization-online-repository.